CHANGES.md

```diff
@@ -78,6 +78,7 @@ To be released.
     instead of the window size of a chunk (i.e., 500).  [[#396], [#399]]
  -  `Swarm<T>.PreloadAsync()` became to get the first parameter, `progress`,
     which accepts `IProgress<PreloadState>`.  [[#397], [#400]]
+ -  Response of `getBlockHashes()` now contains one more higher hash. [[#408]]
```

Comment:
Could you adjust this into like below?

```suggestion
 -  `BlockHashes` messages became to contain one more higher hash. [[#408], [#445]]
```

Also need to a link reference below:

```markdown
[#445]: https://github.com/planetarium/libplanet/pull/445
```

Reply:
I changed changelog. 📃